app/models/manageiq/providers/ibm_terraform/configuration_manager/configuration_profile.rb

```diff
@@ -1,2 +1,6 @@
 class ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfigurationProfile < ::ConfigurationProfile
+  def source_url
```

Comment:
The Lenovo PhysicalInfraManager has a similar concept, their method is `console_url` [[link](https://github.com/ManageIQ/manageiq-providers-lenovo/blob/master/app/models/manageiq/providers/lenovo/manager_mixin.rb#L41-L49)].  There is also a supports_feature_mixin feature that we can use, `supports_console?`

I don't love `console_url` as a name, `provider_url` or something like that would probably be better, but I'd rather keep these concepts similar for now